paddle/framework/scope.cc

```diff
@@ -62,5 +64,17 @@ void Scope::DropKids() {
   kids_.clear();
 }
 
+std::once_flag feed_variable_flag;
+
+framework::Scope* GetScope() {
```

Comment:
There will be more than one scope, so I think `GetGlobalScope()` is a better name.

Reply:
Yes, Exactly